crates/rolldown_binding/src/options/plugin/config/binding_reporter_plugin_config.rs

```diff
@@ -9,6 +9,7 @@ pub struct BindingReporterPluginConfig {
   pub assets_dir: String,
   pub chunk_limit: u32,
   pub should_log_info: bool,
+  pub warn_large_chunks: bool,
```

Comment:
I don't see `vite: reporter` has an option for `warn_large_chunks`. Any reason we don't align with `vite:reporter`?